### PR TITLE
Board page

### PR DESCRIPTION
**Pull request by whirligigY**

### CODE REVIEW

File: src/components/Boards/Boards.jsx
Comment (IgorLap239) [inline src/components/Boards/Boards.jsx:44]: need to remove extra lines after merge

File: src/utils.js
Comment (IgorLap239) [inline src/utils.js:51]: stash console.log

**Review by IgorLap239 — APPROVED** [review]

File: src/components/Boards/Boards.jsx
Comment (Taneros) [inline src/components/Boards/Boards.jsx:8]: resolve convlict

File: src/components/Header/index.jsx
Comment (Taneros) [inline src/components/Header/index.jsx:14]: remove comments

**Comment by Taneros** [comment]
need to pull changes from last PR on main... then create new PR... otherwise do not merge...

**Pull request closed without merging**
